apps/docs/typedoc.config.mjs

```diff
@@ -16,6 +16,7 @@ export default {
   entryPoints: ['../../packages/*'],
   packageOptions: {
     entryPoints: ['src/index.ts'],
+    projectDocuments: ['src/**/*.md'],
```

Comment:
note: projectDocuments needs to be within packageOptions because we're using the "packages" entryPointStrategy